lib/gui/components/safe-webview.js

```diff
@@ -196,16 +196,28 @@ class SafeWebview extends react.PureComponent {
   /**
    * @summary Forward console messages from the webview to analytics module
    * @param {Event} event - event object
+   *
+   * @example
+   * console.log({
```

Comment:
Ahhh, so it'd actually be in the banner page that we do the `console.log` shown here, and `consoleMessage` then forwards that to our analytics services? Now I understand :-)
(so perhaps the example could clarify that?)

Also, would it be worth the analytics-logging clarifying (perhaps by adding an extra property?) that these are messages being generated by whatever JS is running in the page being displayed in the webview, and not messages being generated by the core Etcher application itself?